core/src/main/scala/org/apache/spark/deploy/mesos/MesosClusterDispatcherArguments.scala

```diff
@@ -35,6 +36,11 @@ private[mesos] class MesosClusterDispatcherArguments(args: Array[String], conf:
   propertiesFile = Utils.loadDefaultSparkProperties(conf, propertiesFile)
 
   private def parse(args: List[String]): Unit = args match {
+    case ("--webui-url" | "-u") :: value :: tail =>
```

Comment:
Also -u is not so intuitive (probably it means ui). I would propose just to omit it.